bin/test-db

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+import os
+import sys
+import argparse
+import subprocess
+import time
+from argparse import RawTextHelpFormatter
+
+image_name = "singerio/mysql"
+image_tag = "5.7.29-binlog"
+
+def start_container(name):
+    START_COMMAND = """
+    sudo docker run -d --rm -e "MYSQL_ROOT_PASSWORD={password}" -e "MYSQL_DATABASE={database}" \
+    -p {port}:{port} --name={name} \
+    {image_name}:{image_tag}
+    """.format(password=os.getenv('TAP_MYSQL_PASSWORD'),
+               database=os.getenv('TAP_MYSQL_DBNAME'),
+               port=3306,
+               name=name,
+               image_name=image_name,
+               image_tag=image_tag)
+
+
+    print("Starting Docker process {} using command: {}".format(name, START_COMMAND))
+
+    proc = subprocess.run(START_COMMAND, shell=True)
+    if proc.returncode != 0:
+        sys.exit("Exited with code: {}, the docker process failed to start.".format(proc.returncode))
```

Comment:
Any particular reason to use sys.exit vs just raising an exception which would produce a stack trace?

Reply:
A stack trace will likely just be cluttered since it's a small script. In these cases, running `docker logs mysql1` would be the way to get the actual issue. I can put that in the message here

Reply:
That's fine.